Alteração para modelo hexagonal.

LidianeMendess opened the pull request.

LidianeMendess committed:
Alteração para modelo hexagonal.

Bruuno96 commented on catalogodeprodutos/src/main/java/br/com/cdb/catalogodeprodutos/adapter/output/entity/ProdutoEntity.java:
Pode tirar o import do jakarka.persistence , voce nao esta usando as anotacões.

LidianeMendess replied:
Tabom, vou tirar

LidianeMendess committed:
Port, camada de negócios e pasta infra.

LidianeMendess committed:
Port, camada de negócios e pasta infra.

LidianeMendess committed:
Remove import JakartaPersistence e ajusta GlobalExceptionHandler para arquitetura hexagonal

LidianeMendess committed:
Remoção do JPA e implementação de CRUD via JDBC (JdbcTemplate).

LidianeMendess committed:
- Função SQL para busca por ID integrada via JDBC
- Correção do uso de mappers no projeto
- Adição de logging nos métodos
- Uso de Lombok nas camadas de ponta para simplificar código

LidianeMendess commented:
Refatoração para arquitetura hexagonal com CRUD via JDBC.
